Homework_5/list.py

```diff
@@ -0,0 +1,42 @@
+# 1. Написати програму, яка перевіряє чи є стрінга паліндромом
+palindrome_string_original = input('Write your string here, please: ')  # 'tenet' is a palindrome
+palindrome_string_copy = palindrome_string_original[::-1]  # reverse our string
+
+result = palindrome_string_original == palindrome_string_copy
+
+if result:
+    print(f'String \'{palindrome_string_original}\' is a palindrome')
+else:
+    print(f'String \'{palindrome_string_original}\' is not a palindrome')
+
```

Comment:
Example:
actual_str = input("Enter the string: ")
print(f'Is string {actual_str} palindrome? {actual_str == actual_str[::-1]}')